cmflib/cmf.py

```diff
@@ -1725,5 +1730,30 @@ def non_related_args(type:str,args:dict):
         if repo ==type:
             non_related_args=list(set(available_args)-set(dict_repository_args[repo]))
     return non_related_args
- 
-    
+
+
+def get_python_env()-> str:

```

Comment:
Make this a private function or better still ..add this in utils file

Reply:
done